src/app/static/src/tests/integration/actions.itest.ts

```diff
@@ -4,9 +4,23 @@ import {DynamicFormMeta} from "@reside-ic/vue-dynamic-form";
 
 describe("actions", () => {
 
+    const getStateWithBaselineOptions = async () => {
+        const commit = jest.fn();
+        await (actions[RootAction.FetchPrevalenceGraphConfig] as any)({commit} as any);
```

Comment:
Since this is an integration test, use the actual baseline options defaults when fetching data.